### ISSUE TITLE
[Design] My VA Audit UX Improvements: New Mockups for Saved Applications Section Based on Midpoint Review Feedback

### ISSUE THREAD

**Issue by ericasauv09** [department-of-veterans-affairs/va.gov-team#49390]
## Background

Based on this [revised feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/48655#issuecomment-1302691130) from design platform team we want to create a Mock up of saved applications section as we are trying to find a better way to display the "what benefits does the VA offer" drop down.

## Tasks
- [x] Create mockups
- [x] Explore options and create a recommendation for stacking or side-by-side for multiple cards
- [x] Share designs with team

## Acceptance Criteria
- [x] Mockups for Saved Applications section are created

**Comment by ajakabcin** [comment]
Thanks for creating this @ericasauv09!

@aagosto90 - we pulled this into this sprint. Let's do this as part of step 2 in the sprint [plan notes](https://github.com/department-of-veterans-affairs/va.gov-team/issues/43509#issuecomment-1301185046)

**Comment by aagosto90** [comment]
The description for this ticket is outdated at this point but I updated the Outstanding debts section in the [Sketch file](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/uuid/B3352077-2620-4C5B-9208-2B57CD6CFAC5) so that the cards are side by side on desktop and updated the [FE documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/documentation/outstanding-debts-FE-documentation.md#if-a-user-has-overpayment-debts-and-copay-bills) to reflect that. That's the only update we determined as a team we would be making from Matt D's feedback linked in this ticket. Once Taylor has a FE ticket to implement the change, this ticket can be closed.
